src/main/java/org/xwiki/contrib/macro/docaccordion/internal/DocAccordionMacro.java

```diff
@@ -261,7 +261,7 @@ private DocumentReference getSpaceAWMDataXClass(SpaceReference spaceReference) t
 
         hql.append(excludeQuery);
 
-        hql.append(" AND doc.hidden=0");
+        hql.append(" AND doc.hidden='0'");
```

Comment:
Agree but all I see is: with jetty package the macro working well but not on tomcat + postrgresql database + jndi connector (without quote)... (I open a jirah issue)